test/e2e/yamls/image.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: image
+  namespace: openshift-gitops
+spec:
+  destination:
+    namespace: openshift-gitops
+    server: 'https://kubernetes.default.svc'
+  project: default
+  source:
+    path: argocd/image
+    repoURL: 'https://github.com/openshift/openshift-gitops-examples'
```

Comment:
We should bring the application yaml to this repo instead of depending on an external repo.

Reply:
yes, we can do it. However i think we are using the upstream test repo from openshift org which is a demo example for external user too. Bringing it here will disconnect us from the latest change in upstream demo repo and other advantages.

Reply:
Repeatable test results is important.  Knowing what we are testing in an controlled environment is also important.  I don't see depending on some testing data/manifests that can change without the knowledge of the test is an advantage.

Reply:
make sense 
PTAL - https://github.com/redhat-developer/gitops-operator/commit/25d68cade995ff9e9d71cc29c31fe064c6a3e077